scripts/deploy-federation.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+# This script automates deployment of a federation - as documented
+# in the README - to the current kubectl context.
```

Comment:
Might be helpful to mention that the cluster is also joined into the federation here.

Reply:
Done.